python/glow/gwas/log_reg.py

```diff
@@ -163,7 +220,7 @@ def _logistic_residualize(X: NDArray[(Any, Any), Float], C: NDArray[(Any, Any),
     Residualize the genotype vectors given the null model predictions.
     X_res = X - C(C.T gamma C)^-1 C.T gamma X
     '''
-    X_hat = gwas_fx._einsum('ic,pcd,ds,ps,sg,sp->igp', C, inv_CtGammaC, C.T, gamma, X, Y_mask)
+    X_hat = gwas_fx._einsum('ic,pcd,ds,sp,sg,sp->igp', C, inv_CtGammaC, C.T, gamma, X, Y_mask)
```

Comment:
What happened here?

Reply:
The way that the null model fitting code was written before, it was more convenient for `gamma` to have the shape `n_phenotype x n_sample`. However, on second reading I found that confusing since we typically have one sample per row in the gwas functions. So I transposed `gamma` in the new null fit code.